brave/brave6/src/main/java/com/linecorp/armeria/server/brave/AbstractBraveService.java

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright 2025 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
```

Comment:
Time to update the template?
```suggestion
 * Copyright 2025 LY Corporation
 *
 * LY Corporation licenses this file to you under the Apache License,
```